torch/_inductor/kernel/flex_attention.py

```diff
@@ -628,7 +716,18 @@ def forward_block_mn(
 
     # -- load k --
     # NB reversed order to since K is transposed
-    k = load_checked_block(K_block_ptr, SAFE_HEAD_DIM, IS_DIVISIBLE)
+    if USE_TMA:
+       k = tl._experimental_descriptor_load(  # load in row major
+                desc_k,
+                [start_n.to(tl.int32) , kv_start],
+                [BLOCK_N, QK_HEAD_DIM_ROUNDED],
+                Q.dtype.element_ty,
```

Comment:
do we only need to pass q_along for the dtype? can we update the func to pass that along since I was confused if we needed the base Q again which was suprrising